setup/populate_surveys.rb

```diff
@@ -62,28 +159,66 @@ def populate_survey_answers
   Rapidfire::Answer.insert_all(to_create)
 end
 
-def clear_survey_answers
-  question_group = Rapidfire::QuestionGroup.find_by(name: "Populated Survey")
-  return unless question_group
+def clear_survey_answers 
+  question_group_1 = Rapidfire::QuestionGroup.find_by(name: "generated group 1")
+  question_group_2 = Rapidfire::QuestionGroup.find_by(name: "generated group 2")
```

Comment:
This pattern, creating several variables that are defined separately and have indexes in their names, is something of a code smell. For both readability and extensibility, you could refactor this to use an iterator of some kind, which would make it trivial to change the number of question groups from 2 to something larger later on.